In Fedora, the Linux Libertine font (packaged as linux-libertine-fonts, on which
we depend) is known as Linux Libertine.

Index: gargoyle-fedora/src/garglk/garglk.ini
===================================================================
--- gargoyle-fedora.orig/src/garglk/garglk.ini
+++ gargoyle-fedora/src/garglk/garglk.ini
@@ -52,7 +52,7 @@ lcd           1               # 0=graysc
 monofont      Liberation Mono
 monosize      12.5
 
-propfont      Linux Libertine O
+propfont      Linux Libertine
 propsize      15.5
 
 wmarginx      20              # space around the window frame
Index: gargoyle-fedora/src/garglk/config.c
===================================================================
--- gargoyle-fedora.orig/src/garglk/config.c
+++ gargoyle-fedora/src/garglk/config.c
@@ -54,7 +54,7 @@ float gli_conf_monosize = 12.6;	/* good
 float gli_conf_propsize = 14.7;	/* good size for CharterBT */
 #else
 char *gli_conf_monofont = "Liberation Mono";
-char *gli_conf_propfont = "Linux Libertine O";
+char *gli_conf_propfont = "Linux Libertine";
 float gli_conf_monosize = 12.5;	/* good size for LiberationMono */
 float gli_conf_propsize = 15.5;	/* good size for Libertine */
 #endif
